src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.modifier {
```

Comment:
instead modifier use stars--1 ... stars--5 modifiers and :nth-child pseudo class
for example
`.stars--2 :nth-child(-n+2)`